### PR TITLE
Fix waveform height

### PR DESCRIPTION
**Pull request by dualcnhq**
Screen Shot 2022-06-08 at 04 02 21

Screen Shot 2022-06-08 at 04 02 46

Screen Shot 2022-06-08 at 04 03 46

### CODE REVIEW

File: src/tiles/item-image.ts [hunk 1]
```diff
@@ -26,13 +26,20 @@ export class ItemImage extends LitElement {
   @query('img') private baseImage!: HTMLImageElement;
 
   render() {
+    const width = () => {
+      if (this.isListTile && !this.isCompactTile) return '100px';
```
Comment (mc2) [inline src/tiles/item-image.ts:30]: blocker: the `.mobile` class changes the size of the `<div id="thumb">`
100px -> 90px
45px -> 30px

Comment (mc2) [reply src/tiles/item-image.ts:30]: if `this.isListTile` the width of the waveform is always going to be `100%`

File: src/tiles/item-image.ts [hunk 2]
```diff
@@ -26,13 +26,20 @@ export class ItemImage extends LitElement {
...
+    };
+
     return html`
       <div class=${classMap(this.itemBaseClass)}>
         <img
           class=${classMap(this.itemImageClass)}
           src="${this.imageSrc}"
           alt=""
           @load=${this.onLoad}
+          style="--imgWidth:${width()};"
```
Comment (mc2) [inline src/tiles/item-image.ts:42]: question: if we just get rid of this line, won't the width just work? Or am I missing something?

File: src/tiles/item-image.ts (hunk 1 above)
Comment (dualcnhq) [reply src/tiles/item-image.ts]: fixed it

File: src/tiles/item-image.ts (hunk 2 above)
Comment (dualcnhq) [reply src/tiles/item-image.ts]: i removed it now, thanks for your feedback

File: src/styles/item-image-styles.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@ export const baseItemImageStyles = css`
   .drop-shadow {
     filter: drop-shadow(1px 1px 2px rgba(0, 0, 0, 0.8));
     overflow: hidden;
+    height: 160px;
```
Comment (mc2) [inline src/styles/item-image-styles.ts:10]: blocker: There should not be fixed values in this component. 

Please change to `100%`

**Review by mc2 — CHANGES_REQUESTED** [review]

File: src/styles/item-image-styles.ts (hunk 3 above)
Comment (dualcnhq) [reply src/styles/item-image-styles.ts]: fixed

**Review by mc2 — APPROVED** [review]

**Pull request merged**